### DISCUSSION TITLE
How do you upload Excel File in pynecone?

### DISCUSSION THREAD

**Discussion by basatsu** [reflex-dev/reflex#459] [Q&A] [answered]
I want to upload Excel or CSV and display the data with DataTable component.
Is it possible to do this in Pynecone?
It seems like we can't find upload componet in Pynecone, but do you have any ideas on how to implement this?

**Comment by FHU-yezi** [comment, marked as answer]
Current we can't upload files to server side, but we will able to do this soon.
